### ISSUE TITLE
[Fine-Tuning] there is no document about ViGGO dataset?

### ISSUE THREAD

**Issue by goog** [ray-project/ray#38710]
### Description

i have read the post https://www.anyscale.com/blog/fine-tuning-llama-2-a-comprehensive-case-study-for-tailoring-models-to-unique-applications  
but if i want to try Viggo dataset, is there a guide example?

**Comment by anyscalesam** [comment]
@xwjiang2010 @kouroshHakha is this something we can target as part of next ray release?

**Comment by xwjiang2010** [comment]
@goog Is [this](https://github.com/ray-project/ray/tree/master/doc/source/templates/04_finetuning_llms_with_deepspeed) what you are looking for?

**Comment by goog** [comment]
no, i was looking for the dataset. i have done it https://huggingface.co/googcheng/7b-viggo

**Comment by xwjiang2010** [comment]
Maybe we could publish the cleaned up dataset as an AI here.
